### ISSUE TITLE
Create How-to for using 2019 IPFS improvements in Package Managers

### ISSUE THREAD

**Issue by momack2** [ipfs-inactive/package-managers#94]
Create a quick guide for how to use new 2x faster badger (thanks to @aschmahmann's add improvements) + collaborative clusters (@hsanjuan https://blog.ipfs.io/2020-01-09-collaborative-clusters/) + new Bitswap (@dirkmc) + unixfsv1.5 (from @achingbrain _once this exists in go_) to make your package manager mirror smooth, fast, and collaborative.

@andrew - do you think you could work with these guys to collect the pieces together into a quick "how-to" aimed at package manager mirrors / maintainers to highlight the golden path of adoption hitting our new improvements?

**Comment by andrew** [comment]
@momack2 yeah I think I can slot this in, any particular time frame? Perhaps aligning it with the v0.5 release date?

**Comment by momack2** [comment]
That'd be great! 🙏
